src/mmrelay/matrix/sync_bootstrap.py

```
@@ -0,0 +1,1199 @@
+                                    full_state=False,
+                                    sync_filter=invite_safe_filter,
+                                ),
+                                timeout=facade.MATRIX_SYNC_OPERATION_TIMEOUT,
+                            )
+                        finally:
+                            if original_descriptor is not None:
+                                nio_responses.SyncResponse._get_invite_state = (
+                                    original_descriptor
+                                )
+                            else:
+                                if (
```

Ruff (0.15.9)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)